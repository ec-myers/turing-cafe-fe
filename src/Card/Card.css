.Card {
  background: #ffffff;
  border-radius: 8px;
  border: 4px solid #2c8080;
  color: #2c8080;
  width: 300px;
}

.cancel-btn {
  background: #2c8080;
  color: #ffffff;
  width: 75%;
  margin: 5px;
  height: 25px;
  border-radius: 3px;
  font-size: 14px;
}