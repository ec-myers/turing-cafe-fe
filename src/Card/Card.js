import React from 'react';
import './Card.css';

const Card = (props) => {

  return (
    <div className='Card'>
      <h2>{props.name}</h2>
      <p>{props.date}</p>
      <p>{props.time}</p>
      <p>Number of guests: {props.number}</p>
      <button type='button' className='cancel-btn' onClick={() => props.cancelReservation(props.id)}>Cancel</button>
    </div>
  )
} 

export default Card;