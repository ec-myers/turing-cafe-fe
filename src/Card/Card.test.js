import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

describe('Card', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Card
      name = "Khalid"
      date = "5/9"
      time = "7:30"
      number= {7}
      />);

   expect(wrapper).toMatchSnapshot();
  });


});

