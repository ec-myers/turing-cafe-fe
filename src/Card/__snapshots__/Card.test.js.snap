// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<div
  className="Card"
>
  <h2>
    Khalid
  </h2>
  <p>
    5/9
  </p>
  <p>
    7:30
     pm
  </p>
  <p>
    Number of guests: 
    7
  </p>
  <button
    className="cancel-btn"
    onClick={[Function]}
    type="button"
  >
    Cancel
  </button>
</div>
`;
